### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by juliusvonkohout**
Fix installation instructions

**Checklist:**
- [ ] Unit tests pass:
  **Make sure you have installed kustomize == 3.2.1**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

**Comment by juliusvonkohout** [comment]
@kimwnasptd @jbottum

File: README.md [hunk 1]
```diff
@@ -220,14 +220,15 @@ Install the [Multi-User Kubeflow Pipelines](https://www.kubeflow.org/docs/compon
 ```sh
 kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
 ```
+This installs argo with the safe-to use runasnonroot emissary executor. You still have to make sure yourself that your pipeline main containers
```
Comment (jbottum) [inline README.md:223]: do we need a closing ' after 'runasnonroot ?

File: README.md [hunk 2]
```diff
@@ -220,14 +220,15 @@ Install the [Multi-User Kubeflow Pipelines](https://www.kubeflow.org/docs/compon
...
 
-If your container runtime is not docker, use pns executor instead:
-
+Do not use the deprecated and insecure PNS executor anymore
 ```sh
 kustomize build apps/pipeline/upstream/env/platform-agnostic-multi-user-pns | kubectl apply -f -
 ```
 
-Refer to [argo workflow executor documentation](https://argoproj.github.io/argo-workflows/workflow-executors/#process-namespace-sharing-pns) for their pros and cons.
+Refer to [argo workflow executor documentation](https://argoproj.github.io/argo-workflows/workflow-executors) for furhter reasoning.
```
Comment (jbottum) [inline README.md:231]: I think that further has a typo

File: README.md [hunk 3]
```diff
@@ -220,14 +220,15 @@ Install the [Multi-User Kubeflow Pipelines](https://www.kubeflow.org/docs/compon
 ```sh
 kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
 ```
+This installs argo with the safe-to use runasnonroot emissary executor. You still have to make sure yourself that your pipeline main containers
+are runasnonroot, but at least this fixes it for Argo. Please be aware of the root security risks.
```
Comment (jbottum) [inline README.md:224]: Perhaps rewording (although we could cut sentence #2)....Executing this statement installs Argo with a safe-to-use runasnonroot emissary executor.    Please note that the installer is still responsible  to analyze the security issues opened when containers are run with root access and to decide if the kubeflow pipeline main containers are run as runasnonroot.  It is strongly recommended that the pipelines main containers are installed and run as runasnonroot to mitigate security risks.

**Comment by jbottum** [comment]
let's move this along.

/lgtm

**Comment by kimwnasptd** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**